### ISSUE TITLE
Webtrees-Fancy-ImageBar ver 2.0.10

### ISSUE THREAD

**Issue by madriver** [JustCarmen/webtrees-fancy-imagebar#60]
Hi,
It takes ~ 10 seconds for Webtrees to load with fancy-imagebar enabled.  I believe the issue is this line of code:
 imagecopyresampled($thumb, $source_image, 0, 0, (int)$source_x, (int)$source_y, (int)$thumb_width, (int)$thumb_height, (int)$source_width, (int)$source_height);

From the php-fpm log:
0x00007fd190e24010] imagecopyresampled() /var/www/webtrees2.0/modules_v4/jc-fancy-imagebar/FancyImagebarModule.php:530
		
I have  GD Support | enabled | GD Version | bundled (2.1.0 compatible)
imagick module enabled version 3.44
Webtrees 2.0.11
16GB ram, no errors other that the info in the slow log.  

If I disable FIB site loads instantly.

Thoughts?
Thank you
-Nick

**Comment by JustCarmen** [comment]
This is probably not an issue of the Fancy Imagebar only. If you look at [my website](https://wijzijnfamilie.nl) you will notice the FIB loads instantly. So there are probably other factors causing this. 
- Do you use very large imagefiles on your website? The heavier the images the more effort it will take to manipulate them.  
- Do you notice a difference in loading times when logged in or not? Then privacy calculations might be the issue.

But it is hard to find a cause when I cannot reproduce the issue myself.

**Comment by madriver** [comment]
Same images - Webtrees 1.7 X no issues.  Site loads right off. Webtrees 2.0.11 site takes 10 seconds to load. Without image bar site loads instantly. Is it possible to use imagick instead of GD? At least for testing?  I tried to do a conversion and to say the least it did not work. I am no php programmer.  I get it - this is freeware so to speak.  Your life has more important callings. I disabled the plugin for now. If you make a change and would like me to try it am happy to.  Otherwise  no worries.  Thank you.
